src/databases/Xolotl/avtXolotlFileFormat.C

```diff
@@ -1091,6 +1114,14 @@ avtXolotlFileFormat::GetVar(int timestate, const char *vn)
             rv->SetTuple1(i, zero);
         }
 
+        //Open the network for reading
+        hid_t networkGroup = H5Gopen(fileId, "networkGroup", H5P_DEFAULT);
```

Comment:
I notice you added a "networkGroup" to the class, did you really want to declare this here, since it will end up hiding the one in the class?

Reply:
Add a modification comment to the method header for GetVar.

Reply:
Good catch, this one needed to go.